domains/arnav.json

```diff
@@ -6,7 +6,8 @@
       },
 
       "record": {
-        "A": ["199.36.158.100"]
+        "CNAME": ["arnav-kr.github.io"],
+        "TXT": ["google-site-verification=BIsHFi7TKijw-rS3fXwzHQBATa3PnN0b1b09c0EsdYA"]
```

Comment:
Sorry, we do not support TXT verification, also TXT cannot be an array.

Reply:
Oh I thought you would support it cause there was option to add a TXT record in the web client so i thought it should be work
![IMG_20221007_104112](https://user-images.githubusercontent.com/72879799/194473458-5396767c-5433-435d-9978-ea145d9c33dd.jpg)